src/App.tsx

```diff
@@ -1,8 +1,9 @@
 import React from "react";
-import { usersApi, User } from "./utils/fetcher";
+import { usersApi, User } from "utils/fetcher";
```

Comment:
yang ini bukannya "@/utils/fetcher" ya ?

```suggestion
import { usersApi, User } from "@/utils/fetcher";
```

Reply:
harusnya iya, pake `@/`